packages/networks/bin/keys.js

```diff
@@ -1,10 +1,21 @@
-const { networks } = require('../dist')
+import networks from '../src'
 
 const keys = Object.keys(networks['1'])
```

Comment:
What if we don't have all properties on network 1 though? Usually, we don't deploy any hooks or such first on mainnet.

Reply:
Yeah I was wondering what the best option is, but we actually do deploy everything on mainnet.